data/_TableSchemas.json

```diff
@@ -7193,6 +7244,157 @@
       }
     ]
   },
+  {
+    "Name": "RewardWuest",
```

Comment:
`Wuest` Should this be `Quest`

Reply:
Nice catch. I will fix this typo error :)

Reply:
Fixed:)